webpackConfig/build-min.js

```diff
@@ -0,0 +1,17 @@
+const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
+const webpackMerge = require("webpack-merge");
```

Comment:
They use it throughout the webpack docs: https://webpack.js.org/guides/production-build/#the-manual-way-configuring-webpack-for-multiple-environments and it has features to do "smart" merging of specific webpack properties which might be useful in the future